src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -659,3 +659,14 @@ services:
             - { method: setTranslator, arguments: ['@translator'] }
         tags:
           - { name: form.type }
+
+    prestashop.bundle.form.admin.sell.customer.customer:
+        class: 'PrestaShopBundle\Form\Admin\Sell\Customer\CustomerType'
+        arguments:
+            - '@=service("prestashop.adapter.form.choice_provider.gender_by_id_choice_provider").getChoices()'
+            - '@=service("prestashop.adapter.form.choice_provider.group_by_id_choice_provider").getChoices()'
+            - '@=service("prestashop.adapter.form.choice_provider.risk_by_id_choice_provider").getChoices()'
+            - '@=service("prestashop.core.b2b.b2b_feature").isActive()'
+            - '@=service("prestashop.adapter.legacy.configuration").get("PS_CUSTOMER_OPTIN")'
```

Comment:
Note to myself: test what happens if `PS_CUSTOMER_OPTIN` does not exists. Exception ? Which one ? Raised at runtime or at container building ?

Reply:
it would return default value, in this case `null`.

i dont think we should consider this an issue, do you?